tasklib/tests.py

```diff
@@ -1350,8 +1362,10 @@ def test_symmetric_difference(self):
 
         assert taskset ^ lazyset == set([self.task1, self.task3])
         assert lazyset ^ taskset == set([self.task1, self.task3])
-        assert taskset.symmetric_difference(lazyset) == set([self.task1, self.task3])
-        assert lazyset.symmetric_difference(taskset) == set([self.task1, self.task3])
+        assert taskset.symmetric_difference(lazyset) == set([self.task1,
+                                                             self.task3])
+        assert lazyset.symmetric_difference(taskset) == set([self.task1,
+                                                             self.task3])
```

Comment:
This should be using `self.assertEqual` really, which would make the lines a lot easier to format 😄

Reply:
I think I fixed it in 8f4fabf.

Thank you so much for taking the time of making the review. It served **a
lot**, my contributions from now on will incorporate it ^^

I've also tried to apply the guidelines in the affected files, but I did it
kind of quick so some one might have slipped

Thanks again @robgolding